data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,408 @@ mission "Ringworld Debris: Quarg"
+			`	Using a small instrument similar in appearance to a crowbar, Dusk slowly pulls out a section from the wall, kicking off a good amount of dust in the process. The control station is a two sided panel, full of switches and buttons with small alien markings above each of them.`
+			choice
+				`	"Can you read them?"`
+				`	"Can you open the airlock?"`
+					goto open
+				`	"Are you sure this won't put us in danger?"`
+					goto danger
+			`	"I can, yes. The first row of switches controls the lights inside the complex, the second row controls redundant pressurization systems, and there are similar such operations controlled by the rest of the panel.`
+				goto open
+			label danger
+			`	"There is no way to be certain, but this is the only possible way to get in without destroying half of the complex.`
+			label open
+			`	"This," Dusk says as he presses a switch in the middle of the panel, "is what we were looking for." Soon after, the door in front of you opens, revealing a small room with another door at the opposite end. You follow Dusk inside, where he takes out the scanner again to look for another control panel. Once that is open, another switch allows him to close the door. If it weren't for the lights installed in your helmet, it would now be completely dark. Looking around, you notice that the architecture is distinctly alien, with rounded angles on every side of the room and slides going down off of both doors. The ring itself is too small for you to pass through it standing upright, but the low gravity of the asteroid lets you almost fly though it without much trouble. "We now have to wait for the room to pressurize," he tells you.`
+			choice
+				`	"How long do you think it will it take?"`
+				`	"Would it be safe to remove our helmets after that?"`
+					goto dumb
+			`	"I have no idea. Usually it is a matter of a few minutes, but we know next to nothing about this tech. We will have to wait and see. In the meanwhile," he says as he begins to activate another scanner, "I can analyze the atmosphere that is starting to fill this room. It should save us some time to do it here, as well as being safer, should it be dangerous to our suits."`
+				goto scan
+			label dumb
+			`	Dusk gives you an odd look. "Of course not <first>; it would almost certainly kill you. Such an alien atmosphere is unlikely to be breathable for a human, and even if it was we couldn't be sure until we analyze it extensively." He begins to activate another scanner, "Speaking of which, I can start analyzing the atmosphere that is starting to fill this room. It should save us some time to do it here, as well as also safer, should it be dangerous to our suits."`
```

Comment:
```suggestion
			`	Dusk gives you an odd look. "Of course not <first>; it would almost certainly kill you. Such an alien atmosphere is unlikely to be breathable for a human, and even if it was we could not be sure until we analyze it extensively." He begins to activate another scanner, "Speaking of which, I can start analyzing the atmosphere that is starting to fill this room. It should save us some time to do it here, as well as also safer, should it be dangerous to our suits."`
```